se_mailbox/se_mailbox.py

```diff
@@ -145,9 +145,18 @@ def size(self):
         total_size = 0
         total_count = 0
         for i, line in enumerate(open(self.size_fn).readlines()[1:]):
-            size, count = line.split()
-            total_size += int(size)
-            total_count += int(count)
+            if not line or len(line.split()) != 2:
```

Comment:
I don't understand why we changed these lines.

Reply:
Seems that we want to make sure that each processed line is in the correct format (having 2 values, no extra spaces and the correct type) in order to avoid errors.

Reply:
That doesn't seem like a Python 2->3 change?